### ISSUE TITLE
multimc: 0.6.14

### ISSUE THREAD

**Issue by lourkeur** [NixOS/nixpkgs#149923]
https://github.com/MultiMC/Launcher/blob/70d400f205952912cbfa0a13cd8c84687ef06baa/changelog.md#multimc-0614

###### Checklist

- [x] Checked the [nixpkgs master branch](https://github.com/NixOS/nixpkgs)

- [x] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

###### Project name
`nix search` name:

current version: unstable (technically 0.6.13)
desired version: 0.6.14

###### Notify maintainers

maintainers: @starcraft66 @cleverca22

###### Note for maintainers

Please tag this issue in your PR.

**Comment by huantianad** [comment]
Out of curiosity, with the branding changes, would we use our own assets or are we still going to use the "official" assets.

A Minecraft dirt block with the Nix logo would be a kinda funny alternative :stuck_out_tongue:

**Comment by starcraft66** [comment]
I plan to use the default unbranded assets for the initial update but we can work out our own assets for sure.

**Comment by starcraft66** [comment]
This is currently blocked by https://github.com/MultiMC/Launcher/issues/4328 because the latest version of MultiMC can no longer be launched from the read-only Nix store.

**Comment by starcraft66** [comment]
Draft PR is up here, just waiting for upstream https://github.com/NixOS/nixpkgs/pull/150285

**Comment by thelegy** [comment]
As upstream seems to make no movement to address the issue, maybe we can fix it on our end. (The upstream issue was closed as "Working as intended")
https://github.com/MultiMC/Launcher/issues/4328#issuecomment-991923874 suggests the `-d` or better `--dir` option would fix the issue. Can we set it in a wrapper and point it to something like `${XDG_DATA_HOME:-~/.local/share}/multimc`?

**Comment by huantianad** [comment]
@thelegy I believe this is mentioned and implemented over in the [PR](https://github.com/NixOS/nixpkgs/pull/150285#pullrequestreview-829644371).

**Comment by starcraft66** [comment]
I have been made aware of the existance of a new MultiMC fork created due to the hostility of MultiMC's developer towards maintainers of 3rd party packages: https://github.com/PolyMC/PolyMC

I think it might be worthwhile to package this and see where that fork goes given the backwards-compatibility issues the de-branded launcher currently pose. Perhaps the "logging to the wrong location bug" MultiMC's maintainers refused to fix could be patched in PolyMC at the same time.

**Comment by starcraft66** [comment]
I have opened the following issue: https://github.com/PolyMC/PolyMC/issues/23

**Comment by legendofmiracles** [comment]
Another PR that would update multimc: https://github.com/NixOS/nixpkgs/pull/150285

**Comment by starcraft66** [comment]
New PR open: https://github.com/NixOS/nixpkgs/pull/154051/files

**Comment by thelegy** [comment]
As #154051 was merged, which also removes multimc from nixpkgs, I would like to close this.
